simple_test_open_channel.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
```

Comment:
This whole file should probably be in the tests directory.

Reply:
I moved the file to the tests directory.